File: app/src/main/java/com/jokysss/mirror/HomeActivity.java
package com.jokysss.mirror;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class HomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);
        findViewById(R.id.bt_view).setOnClickListener(v -> startActivity(new Intent(HomeActivity.this, MainActivity.class)));
        findViewById(R.id.bt_scroll).setOnClickListener(v -> startActivity(new Intent(HomeActivity.this, ScrollActivity.class)));
        findViewById(R.id.bt_page).setOnClickListener(v -> startActivity(new Intent(HomeActivity.this, PageScrollActivity.class)));
        findViewById(R.id.bt_list).setOnClickListener(v -> startActivity(new Intent(HomeActivity.this, ListViewActivity.class)));
        findViewById(R.id.bt_slide_menu).setOnClickListener(v -> startActivity(new Intent(HomeActivity.this, SlideMenuActivity.class)));
        findViewById(R.id.bt_swipe_menu).setOnClickListener(v -> startActivity(new Intent(HomeActivity.this, SwipeMenuListViewActivity.class)));

    }
}

File: app/src/main/java/com/jokysss/mirror/HomeActivity.kt
package com.jokysss.mirror

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_home.*

class HomeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)
        bt_view.setOnClickListener { startActivity(Intent(this@HomeActivity, MainActivity::class.java)) }
        bt_scroll.setOnClickListener { startActivity(Intent(this@HomeActivity, ScrollActivity::class.java)) }
        bt_page.setOnClickListener { startActivity(Intent(this@HomeActivity, PageScrollActivity::class.java)) }
        bt_list.setOnClickListener { startActivity(Intent(this@HomeActivity, ListViewActivity::class.java)) }
        bt_slide_menu.setOnClickListener { startActivity(Intent(this@HomeActivity, SlideMenuActivity::class.java)) }
        bt_swipe_menu.setOnClickListener { startActivity(Intent(this@HomeActivity, SwipeMenuListViewActivity::class.java)) }
        bt_tab_icon.setOnClickListener { startActivity(Intent(this@HomeActivity, TabIconActivity::class.java)) }
        bt_test.setOnClickListener { startActivity(Intent(this@HomeActivity, TestActivity::class.java)) }
    }
}
